### PR TITLE
My PR Fixes FRO-205 - change organization time zone

### PR DESCRIPTION
**Pull request by rambey-coder**
# Fixes FRO-205 - change organization time zone
# Fixes FRO-206 - change organization language

( This could be an existing issue or a linear ticket )

My PR Fixes FRO-205 - change organization time zone
My PR Fixes FRO-206 - change organization language
Linear Ticket link - https://linear.app/team-brainbox/issue/FRO-205/zuri-chat-change-timezone
Linear Ticket link - https://linear.app/team-brainbox/issue/FRO-206/zuri-chat-change-language

# Changes proposed

## What were you told to do?
Allow organization to change time zone
Allow organization to change language

## What did you do?

- Check if there is an admin user logged.
- Get the User_id and the organization_id.
- Use these details to change and update the organization time zone and language in admin setting.
- set the updated time zone or language to storage.
- Make the patch request to the API backend.

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos
Capturex
Capture

### CODE REVIEW

File: packages/main/src/components/time-zone/TimeZone.jsx [hunk 1]
```diff
@@ -1,12 +1,58 @@
...
+      setUser(user);
+    }
+  }, []);
+
+  const handleTimezone = e => {
+    e.preventDefault();
+
+    axios
+      .patch(
+        `https://api.zuri.chat/organizations/${organization_id}/members/${user_member_id}/settings`,
```
Comment (JulianaSau) [inline packages/main/src/components/time-zone/TimeZone.jsx:29]: Import BASE_API_URL from the utilities package in zc_main

File: packages/main/src/components/time-zone/TimeZone.jsx [hunk 2]
```diff
@@ -1,12 +1,58 @@
...
+            Authorization: "Bearer " + user.token
+          }
+        }
+      )
+      .then(res => {
+        const userTimeZone = JSON.parse(sessionStorage.getItem("user"));
+
+        userTimeZone.time_zone = selectTimezone;
+
+        sessionStorage.setItem("user", JSON.stringify(userTimeZone));
```
Comment (JulianaSau) [inline packages/main/src/components/time-zone/TimeZone.jsx:47]: You can also add a toast notification to signify that the change has successfully been made

**Review by JulianaSau — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -47,5 +46,7 @@
     "rimraf": "^3.0.2",
     "serve": "^12.0.0"
   },
-  "dependencies": {}
```
Comment (billmal071) [inline package.json:50]: There's already a toast library installed and this is the wrong place to install it.

File: packages/main/src/components/language/Language.jsx [hunk 4]
```diff
@@ -1,13 +1,53 @@
...
+
+  // let organization_id = JSON.parse(sessionStorage.organisations)[0].id;
+  // let user_member_id = JSON.parse(sessionStorage.organisations)[0].member_id;
+
+  // useEffect(() => {
+  //   const user = JSON.parse(sessionStorage.getItem("user"));
+  //   if (user) {
+  //     setUser(user);
+  //   }
+  // }, []);
```
Comment (billmal071) [inline packages/main/src/components/language/Language.jsx:19]: Remove commented out code.

File: packages/main/src/components/language/Language.jsx [hunk 5]
```diff
@@ -1,13 +1,53 @@
...
+    //   },
+
+    //   {
+    //     headers: {
+    //       Authorization: "Bearer " + user.token
+    //     }
+    //   }
+    // );
+  };
+  console.log(selectLanguage);
```
Comment (billmal071) [inline packages/main/src/components/language/Language.jsx:44]: remove commented out code and console.log statements.

**Review by billmal071 — CHANGES_REQUESTED** [review]

**Comment by billmal071** [comment]
@rambey-coder

File: packages/main/src/components/language/Language.jsx (hunk 4 above)
Comment (rambey-coder) [reply packages/main/src/components/language/Language.jsx]: resolved

**Review by JulianaSau — APPROVED** [review]

**Comment by JulianaSau** [comment]
Hey @rambey-coder, resolve conflicts so that I can merge😅

**Pull request merged**